src/NuGet.Services.BasicSearch/NuGet.Services.BasicSearch.csproj

```diff
@@ -16,10 +16,10 @@
     <AssemblyName>NuGet.Services.BasicSearch</AssemblyName>
     <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
     <UseIISExpress>true</UseIISExpress>
-    <IISExpressSSLPort />
-    <IISExpressAnonymousAuthentication />
-    <IISExpressWindowsAuthentication />
-    <IISExpressUseClassicPipelineMode />
+    <IISExpressSSLPort>25225</IISExpressSSLPort>
```

Comment:
Does this keep getting adding by the IDE if you revert it? If not, let's revert this stuff.

Reply:
Nice catch!

Reply:
But otherwise, yeah, it was added by IDE automagically when I updated the package version through the UI.